libs/secure/util.hpp

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Do we need these utils? Looks like they are not used at all. At least for this PR.

Reply:
They were used previously, but after cleanup -- not used. 
However, it is not so much code for them and they will be useful when I start next pr.
Pay attention at the base branch of this PR.